examples/test-merge-geometry/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Cube</title>
+    <meta name="description" content="Merge Geometry - A-Frame">
+    <script src="../../dist/aframe.js"></script>
+    <style>
+      body { background-color: black; }
+      .stats {
+        position: fixed;
+        text-align: center;
+        top: 200px;
+        right: calc(50% - 200px);
+        left: calc(50% - 200px);
+        width: 400px;
+        height: 50px;
+        color: white;
+        padding: 10px;
+        font-size: 16pt;
+        line-height: 1.2em;
+        background-color: #EF2D5E;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="stats"></div>
+    <a-scene stats="true">
+      <a-assets>
+        <a-mixin id="cube"
+                 geometry="primitive: box; height: 16; width: 16; depth: 16; mergeTo: #boxes; skipCache: true; buffer: false"
+                 material="color: #167341; roughness: 1.0; metalness: 0.2;"></a-mixin>
+      </a-assets>
+      <a-entity position="0 20 0">
+        <a-entity camera="active:true" wasd-controls="" look-controls="" rotation="0 0 0" position="0 0 0"></a-entity>
+      </a-entity>
+      <a-entity id="boxes" mixin="cube" geometry="height: 0"></a-entity>

```

Comment:
`height: 0` is a bit awkward if you want to merge to a container.

we should probably just allow not specifying a "primitive" to just instantiate a blank `new THREE.Geometry()`.